presto-main/src/test/java/io/prestosql/security/TestConnectorAccessControlForProcedures.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.security;
+
+import io.prestosql.spi.connector.ConnectorAccessControl;
+import org.testng.annotations.Test;
+
+import static io.prestosql.spi.testing.InterfaceTestUtils.assertAllMethodsOverridden;
+
+public class TestConnectorAccessControlForProcedures
+{
+    @Test
+    public void testEverythingImplemented()
+    {
+        assertAllMethodsOverridden(ConnectorAccessControl.class, ConnectorAccessControlForProcedures.class);
```

Comment:
Can we also use `io.prestosql.spi.testing.InterfaceTestUtils#assertProperForwardingMethodsAreCalled(java.lang.Class<I>, java.util.function.Function<I,C>)`?

Or do something similar?